handlers/utils/utils.go

```diff
@@ -182,47 +165,111 @@ func tempFile(destination string) string {
 	return ""
 }
 
-func downloadFromURL(rawurl string, filepath string) error {
-	file, err := os.Open(filepath)
-	if err == nil {
-		response, err1 := http.Get(rawurl)
-		if err1 != nil {
-			logrus.Error(fmt.Sprintf("Error while downloading error: %s", err1))
-			return err1
+func GetResponseData(event *events.Event) map[string]interface{} {
+	resourceType := event.ResourceType
+	switch resourceType {
+	case "instanceHostMap":
+		return map[string]interface{}{resourceType: getInstanceHostMapData(event)}
+	case "volumeStoragePoolMap":
+		return map[string]interface{}{
+			resourceType: map[string]interface{}{
+				"volume": map[string]interface{}{
+					"format": "docker",
+				},
+			},
 		}
-		defer response.Body.Close()
-		n, ok := io.Copy(file, response.Body)
-		if ok != nil {
-			logrus.Error(fmt.Sprintf("Error while copying file: %s", ok))
-			return ok
+	case "instancePull":
+		return map[string]interface{}{
+			"fields": map[string]interface{}{
+				"dockerImage": getInstancePullData(event),
+			},
 		}
-		logrus.Info(fmt.Sprintf("%v bytes downloaded successfully", n))
-		return nil
+	default:
+		return map[string]interface{}{resourceType: map[string]interface{}{}}
```

Comment:
based on the python version, I _think_ this should just return an empty map, no inner type.